build_runner_core/lib/src/generate/options.dart

```diff
@@ -19,8 +19,22 @@ import '../package_graph/target_graph.dart';
 import '../util/hash.dart';
 import 'exceptions.dart';
 
+/// The default list of files visible for non-root packages.
+///
+/// This is also the default list of files for targets in non-root packages when
+/// an explicit include is not provided.
+const List<String> defaultNonRootVisibleAssets = [
```

Comment:
@natebosch particularly it might be good to get another set of eyes here - this list seems pretty reasonable to me though.

Possibly we should add `'CHANGELOG*'`?

Reply:
This list looks fine to me. I'd be in favor of adding `CHANGELOG*`